components/MobileNav.tsx

```diff
@@ -20,122 +21,132 @@ import {
   NavbarLinkWhyUse
 } from "./NavbarComponents"
 
-const NavBarBoxContainer: FC<
-  React.PropsWithChildren<{ className?: string }>
-> = ({ children, className }) => {
-  return (
-    <div
-      className={`d-flex flex-row align-items-start justify-content-between w-100`}
-    >
-      {children}
-    </div>
-  )
-}
+export const MobileNav: React.FC<React.PropsWithChildren<unknown>> = () => {
+  const BlackCollapse = styled(() => {
+    return (
+      <Navbar.Collapse id="basic-navbar-nav" className="bg-black mt-2 ps-4">
+        {/* while MAPLE is trying to do away with inline styling,   *
```

Comment:
Yeah, sadly setting specific heights aren't covered by bootstrap. Weird that styled-components aren't respecting that - not worth digging into here, but thanks for the heads up to look out for this issue in the future.

That said, why do we need to set a specific height here? Shouldn't the menu just be the height of the links list, whichever set of links we're dealing with?